src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/_help.py

```diff
@@ -10,4 +10,12 @@
 helps['configure'] = """
             type: command
             short-summary: Configure Azure CLI 2.0 Preview or view your configuration. The command is interactive, so just type `az configure` and respond to the prompts.
+            examples:
+                - name: configure a default resource group
+                  text: >
+                    az configure --setting resource_group myRG
+                - name: configure a default webapp
+                  text: >
+                    az configure --section compute/vm --value myvm
```

Comment:
I thought about that yesterday as well. 
For this feature, there is no good discoverability, because the whole process is opt-in, so you will need to check out the help on the argument to know whether this arg's default value can be configured. That is why I initially exposed a strong typed command of `set-defaults` 
That said, I still think we can do some change to get closer to common sense. Right now 'resource_group' goes to `[core]` section and "appservice/webapp" goes to `[appservice]` section, I am thinking maybe we can just use a section like `[default]`, and put in all those values there? This way, it is fine to just use `group`